ui/desktop/app/controllers/scopes/scope/projects/sessions/index.js

```diff
@@ -15,7 +16,8 @@ export default class ScopesScopeProjectsSessionsIndexController extends Controll
    */
   get sorted() {
     const sessions = this.model;
-    const sortedSessions = sessions
+    if (sessions.length <= 1) return sessions;
+    const sortedSessions = A(sessions)
```

Comment:
I encountered an error when filtering sessions. Seems like using an ember array fixes that, im assuming because `sortBy` is a method particular to ember ?
![image](https://github.com/hashicorp/boundary-ui/assets/107949262/545b8327-a705-427b-822b-70834e8b19e7)

Reply:
That's surprising, do you know at what point it's returning a normal array instead of ember array from the model? I'm curious how this was working before

Reply:
Does an ember array `.filter` not return it as another ember array? I'm just trying to understand where it becomes a normal array and whether we should actually be converting to an ember array upstream

Reply:
I might do the actual converting in the `model` hook since the expectation might be that it should have already been an ember array for the consumers, any thoughts?

Reply:
Yes, I agree, and thats a good point! Updated!